app/components/views/Info/CopyAddressButton/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from "@emotion/css";
+
+// used !important here bc home component global css is more specific than this one
+// idk what this global was doing .copyAddressButtonIcon:global(.MuiSvgIcon-root) {
```

Comment:
This was targeting a classname which is created by Mui
the :global() tells modules not to apply a random scoped classname to the selector, as random scoped classnames are the default behavior (to prevent style clashes when reusing the component).

So in emotion land I think the equivalent is this:

```ts
export const copyAddressButtonIcon = css`
  .MuiSvgIcon-root {
    height: 1.6rem;
    width: 1.6rem;
    cursor: pointer;
  }
`;
```

Reply:
The `Home_container svg` class is more specific than the .MuiSvgIcon-root class. I think when we update home component to use emotion css and ```import { injectGlobal } from '@emotion/css'

injectGlobal`
  * {
    box-sizing: border-box;
  }``` this injectGlobal it might fix the specificity.